python/ray/ray_constants.py

```diff
@@ -130,6 +130,7 @@ def to_memory_units(memory_bytes, round_up):
 DASHBOARD_AGENT_DIED_ERROR = "dashboard_agent_died"
 DASHBOARD_DIED_ERROR = "dashboard_died"
 RAYLET_CONNECTION_ERROR = "raylet_connection_error"
+PLACEMENT_GROUP_PUSH_ERROR = "placement_group_error"
```

Comment:
```suggestion
PLACEMENT_GROUP_PUSH_ERROR = "placement_group"
```

Reply:
should be place_group_error?

Reply:
I found the others don't have `error`, for example:`REGISTER_ACTOR_PUSH_ERROR`. @rkooo567 What's your suggestion?

Reply:
Yeah naming conventions here are not that consistent in general... I prefer to follow the majority (no error).

Reply:
updated, also updated `raylet_connection_error`